notebooks/analyze_looping_across_lending_protocols.ipynb

```diff
@@ -75,12 +75,12 @@
     "    query = f\"SELECT * FROM {protocol}\"\n",
     "    df = pd.read_sql_query(query, conn)\n",
     "    \n",
-    "    pd.set_option('display.max_columns', None)  # Display all columns\n",
-    "    pd.set_option('display.max_colwidth', None)  # Display full column width\n",
-    "    pd.set_option('display.width', None)  # Adjust display width\n",
+    "pd.set_option('display.max_columns', None)  # Display all columns\n",
+    "pd.set_option('display.max_colwidth', None)  # Display full column width\n",
+    "pd.set_option('display.width', None)  # Adjust display width\n",
```

Comment:
You still set these in multiple cells. It suffices to set it just once.

Reply:
Done! ✅